src/main/java/io/appium/java_client/flutter/commands/DoubleClickParameter.java

```diff
@@ -0,0 +1,34 @@
+package io.appium.java_client.flutter.commands;
+
+import com.google.common.base.Preconditions;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@Accessors(chain = true)
+@Setter
+@Getter
+public class DoubleClickParameter extends FlutterCommandParameter {
+    private WebElement element;
+    private Point offset;
+
+
+    @Override
+    public Map<String, Object> toJson() {
+        Preconditions.checkArgument(element != null || offset != null,
+                "Must supply a valid element or offset to perform flutter gesture event");
```

Comment:
please avoid using Guava as it is going to be dropped by Selenium, and it makes sense to drop its usage by Appium Java client

Reply:
Sounds good. I'll replace the checks with `import org.openqa.selenium.internal.Require`